torch/profiler/_pattern_matcher.py

```diff
@@ -603,8 +631,31 @@ def report_all_anti_patterns(prof, should_benchmark: bool = False, json_report_d
             if report_msg not in reported:
                 message_list.append(report_msg)
                 reported.add(report_msg)
+                src_location, line_no = source_code_location(event).split(":")
+                report_dict.setdefault(src_location, []).append({
+                    "line_number":
```

Comment:
nit: kind of strange indentation. How come it isn't a key and value per line?

Reply:
Kinda overly relying on auto format. I will clean that up :)